actionmetadata_test.go

```diff
@@ -35,6 +39,15 @@ func TestKnowledgeBase(t *testing.T) {
 				lintIssues = append(lintIssues, fmt.Sprintf("File must be named action-security.yml, not %s at %s", info.Name(), filePath))
 				return nil
 			}
+
+			// validating the action repo
+			var response *github.Response = doesActionRepoExist(filePath)
```

Comment:
`doesActionRepoExist` should return a bool. The calling method should not need to know the details of how you are checking this, so don't return status code.